test/unit/BuildSelectSqlCapableTraitTest.php

```diff
@@ -174,7 +175,18 @@ public function testBuildSelectSql()
                 ->with($joinConditions, $valueHashMap)
                 ->willReturn($joins);
 
-        $expected = "SELECT $columnsList $from $joins $where;";
+        $grouping = [
+            uniqid('field1'),
+            uniqid('field2'),
+            uniqid('field3'),
+        ];
+        $groupBy = 'GROUP BY field1, field2, field3';
```

Comment:
The code seems to quote identifiers. How come the field names pass the test while being unquoted?

Reply:
Because I'm mocking the code, no quoting is actually happening.